actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -16,8 +16,22 @@ module Rendering
     extend ActiveSupport::Concern
     include ActionView::ViewPaths
 
-    # Normalizes arguments, options and then delegates render_to_body and
+    # Normalizes arguments, options and then delegates <tt>render_to_body</tt> and
     # sticks the result in <tt>self.response_body</tt>.
+    #
+    # The primary options are:
```

Comment:
I feel this documentation is on the wrong place. `AbstractController::Rendering#render` don't know anything about `partial` for example. All those options are from `ActionController::Rendering#render`, so this documentation should be there.

While `AbstractController::Rendering#render` know about `:html`, it should not and this is an technical debt.

About linking to the guides I'm against of it. We do not link to guides in any of our documentation. The documentation should be self-contained.